package.json

```diff
@@ -35,30 +35,30 @@
   },
   "homepage": "https://github.com/langpavel/node-pg-async#readme",
   "dependencies": {
-    "bluebird": "^3.4.6",
-    "debug": "^2.6.3",
-    "pg": "^6.1.4"
+    "bluebird": "3.5.0",
+    "debug": "2.6.3",
+    "pg": "6.4.0"
```

Comment:
It is not such import, you can revert to `^`. I now semver is working, but with certain version packages, i'm more sure, that nothing break.